ext/dba/tests/gh16990.phpt

```diff
@@ -0,0 +1,46 @@
+--TEST--
+GH-16990 (dba_list() is now zero-indexed instead of using resource ids)
+--EXTENSIONS--
+dba
+--CONFLICTS--
+dba
+--SKIPIF--
+<?php
+$handler = "flatfile";
+require_once(__DIR__ .'/skipif.inc');
```

Comment:
See https://github.com/php/php-src/pull/17011#discussion_r1865806461 f.

Reply:
Thanks for pointing this out!